core/config/docs/chains-evm.toml

```diff
@@ -399,6 +399,9 @@ MaxAllowedFinalityDepth = 10000 # Default
 # On chains with fast finality, the persistence layer does not improve the chain's load time and only consumes database resources (mainly IO).
 # NOTE: persistence should not be disabled for products that use LogBroadcaster, as it might lead to missed on-chain events.
 PersistenceEnabled = true # Default
+# PersistenceBatchSize is used to batch head tracker db transactions (inserts and deletes).
+# Default is set to 100, which will be the max number of heads the head tracker will need to pull from rpc on startup (if the batch is at 99 and head tracker crashes)
```

Comment:
`if the batch is at 99 and head tracker crashes` could you elaborate in the comment? You mean that it crashes if `PersistenceBatchSize` is set to `99` or that it crashes if it is set to `100`, but fetches 99?